applications/luci-app-aria2/htdocs/luci-static/resources/view/aria2/log.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+'require dom';
+'require fs';
+'require poll';
+'require uci';
+'require view';
+
+var css = '				\
+#log_text {				\
+	padding: 10px;			\
+	text-align: left;		\
+}					\
+#log_text pre {				\
+	padding: .5rem;			\
+	word-break: break-all;		\
+	margin: 0;			\
+}					\
+.description {				\
+	background-color: #33ccff;	\
+}					\
+';
+
+function pollLog(e) {
+	return uci.load('aria2')
+		.then(function() {
+			var logFile = uci.get_first('aria2', 'aria2', 'log') || '/var/log/aria2.log';
+			return Promise.all([
+				fs.exec_direct('/usr/bin/tail', [ '-n', '50', logFile ]).then(function(res) { return res.trim().split(/\n/).reverse().join('\n') }),
```

Comment:
This is a matter of taste, but I think the lines are too long. Can you please reformat them?

Reply:
Done.